### PR TITLE
Feature/#228 style dropdown options

### PR DESCRIPTION
**Pull request by sivxkxn**
Adding styles to the dropdown with categories.

### CODE REVIEW

File: src/app/shared/styles/create-form.scss [hunk 1]
```diff
@@ -143,4 +143,31 @@ button:focus {
   font-weight: normal;
   font-size: 13px;
   color: #333333;
-} 
\ No newline at end of file
+  
+} 
+ ::ng-deep .mat-option-text{
```
Comment (dmitryy90) [inline src/app/shared/styles/create-form.scss:148]: indentations

Comment (dmitryy90) [inline src/app/shared/styles/create-form.scss:148]: use some namespace class to avoid overriding other dropdowns

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/styles/create-form.scss [hunk 2]
```diff
@@ -143,4 +143,27 @@ button:focus {
...
+    font-size: 13px;
+    color: #000000;
+    background-color : white;
+    padding-left: 10px;
+    &:hover{
+      background-color : #F8F8F8;
+    }
+   }
+   .mat-option{
+    padding: 0;
```
Comment (dmitryy90) [inline src/app/shared/styles/create-form.scss:162]: indentations

File: src/app/shared/styles/create-form.scss [hunk 3]
```diff
@@ -143,4 +143,27 @@ button:focus {
   font-weight: normal;
   font-size: 13px;
   color: #333333;
-} 
\ No newline at end of file
+  
+} 
+::ng-deep #mat-select-6-panel {
```
Comment (dmitryy90) [inline src/app/shared/styles/create-form.scss:148]: not sure about this selector. seems it auto generated.

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/category-select/category-select.component.html [hunk 4]
```diff
@@ -1,21 +1,21 @@
-<form [formGroup]="CategoryFormGroup">
-  <mat-select class="step-input" formControlName="category" (selectionChange)="onSelectCategory($event.value)"
+<form [formGroup]="CategoryFormGroup" class="form-category">
```
Comment (litvinets) [inline src/app/shared/components/category-select/category-select.component.html:1]: why do you need this class?

**Comment by litvinets** [comment]
Unfortunately, you still have mismatching with the design.
The expected result: 
image
The actual result:
image

1. Remove please the gray color on the selected option on both sides and the division line between options.
2. The dropdown for the first option should not cover the selection input, it should be moved below. The other two options work correctly. 
image

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]

**Pull request merged**